components/script/dom/htmliframeelement.rs

```diff
@@ -347,6 +349,19 @@ impl MozBrowserEventDetailBuilder for HTMLIFrameElement {
                     canGoForward: Some(can_go_forward),
                 }.to_jsval(cx, rval);
             }
+            MozBrowserEvent::OpenTab(url, target) => {
```

Comment:
I'm ok with leaving that here, but this is not used yet (this will need to be fired when the user ctrl/cmd/middle-clicks on a link). And in this case, there is no target.